src/promptflow/promptflow/_sdk/_orm/trace.py

```diff
@@ -0,0 +1,43 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from sqlalchemy import TEXT, Column, Index
+from sqlalchemy.orm import declarative_base
+
+from promptflow._sdk._constants import SPAN_TABLENAME
+
+from .retry import sqlite_retry
+from .session import trace_mgmt_db_session
+
+Base = declarative_base()
+
+
+class Span(Base):
+    __tablename__ = SPAN_TABLENAME
+
+    span_id = Column(TEXT, primary_key=True)
+    trace_id = Column(TEXT)
+    parent_id = Column(TEXT, nullable=True)
+    # we might not need `experiment_name` if we have partition,
+    # then the file name should be `experiment_name` or "default"
+    experiment_name = Column(TEXT, nullable=True)
+    run_name = Column(TEXT, nullable=True)
+    path = Column(TEXT)
+    content = Column(TEXT)  # JSON string
```

Comment:
THhere might be requirements to fileter by span type & function name:

![image](https://github.com/microsoft/promptflow/assets/47586720/7117ac07-e337-4107-9613-500907ccb9e0)